src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -197,28 +197,28 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
 
                     pathEndOrSingleSlash {
                       getPolicy(policyId, userInfo, samRequestContext) ~
-                      putPolicyOverwrite(resourceType, policyId, userInfo, samRequestContext) ~
-                      deletePolicy(policyId, userInfo, samRequestContext) ~
-                      pathPrefix("memberEmails") {
-                        requireActionsForSharePolicy(policyId, userInfo, samRequestContext) {
-                          pathEndOrSingleSlash {
-                            putPolicyMembershipOverwrite(resourceType, policyId, userInfo, samRequestContext)
-                          } ~
-                          pathPrefix(Segment) { email =>
-                            withSubject(WorkbenchEmail(email), samRequestContext) { subject =>
-                              pathEndOrSingleSlash {
-                                putUserInPolicy(policyId, subject, samRequestContext) ~
-                                deleteUserFromPolicy(policyId, subject, samRequestContext)
-                              }
+                        putPolicyOverwrite(resourceType, policyId, userInfo, samRequestContext) ~
+                        deletePolicy(policyId, userInfo, samRequestContext)
+                    } ~
```

Comment:
the structure was broken here leaving the `memberEmails` routes below out of place